legend-engine-pure-code-compiled-core/src/main/resources/core/pure/mutation/mutation.pure

```diff
@@ -0,0 +1,16 @@
+import meta::pure::mapping::*;
+import meta::pure::runtime::*;
+import meta::pure::graphFetch::*;
+
+function <<functionType.NotImplementedFunction>> meta::pure::mutation::save<T>(values: T[*], rootGraphFetchTree: RootGraphFetchTree<T>[1], mapping: Mapping[1], runtime: Runtime[1]): T[*]

```

Comment:
you can add this stereotype to avoid preeval explicitly 

https://github.com/finos/legend-engine/pull/1765/files#diff-58e186f6e55f338c32a49ce3d1db33c32db4f0cd030a29700f0f26f63a94eaaaR148